CRM/Eck/Page/EntityTypes.php

```diff
@@ -18,10 +18,9 @@
 class CRM_Eck_Page_EntityTypes extends CRM_Core_Page {
 
   public function run() {
-    // Example: Set the page-title dynamically; alternatively, declare a static title in xml/Menu/*.xml
     CRM_Utils_System::setTitle(E::ts('ECK Entity Types'));
 
-    $entity_types = civicrm_api3('EckEntityType', 'get', [], ['limit' => 0])['values'];
```

Comment:
FYI @jensschuppe I've noticed this mistake in a lot of the APIv3 calls in this extension: `['limit' => 0]` is in the wrong place and therefore wouldn't have worked. This would have caused problems if anyone ever made > 25 entity types.